pr_testing/test_multiple_prs.sh

```diff
@@ -1181,23 +1181,37 @@ for BT in ${ENABLE_BOT_TESTS}; do
              pushd $WORKSPACE/$CMSSW_IB/src/$PROFILING_WORKFLOW
              ./profile.sh $CMSSW_IB || true
              get_jenkins_artifacts igprof/${CMSSW_IB}/${ARCHITECTURE}/profiling/${PROFILING_WORKFLOW}/RES_CPU_step3.txt  ${CMSSW_IB}_RES_CPU_step3.txt || true
-             $WORKSPACE/profiling/Analyze_tool/compare_cpu_txt.py --old ${CMSSW_IB}_RES_CPU_step3.txt --new RES_CPU_step3.txt > RES_CPU_compare.txt || true
+             $WORKSPACE/profiling/Analyze_tool/compare_cpu_txt.py --old ${CMSSW_IB}_RES_CPU_step3.txt --new RES_CPU_step3.txt > RES_CPU_compare_$PROFILING_WORKFLOW.txt || true
+             mkdir -p $WORKSPACE/upload/profiling/
+             cp -p RES_CPU_compare_$PROFILING_WORKFLOW.txt $WORKSPACE/upload/profiling/$d/ || true
              popd
              pushd $WORKSPACE/$CMSSW_IB/src || true
-             for f in $(find TimeMemory -type f -name '*.sql3' -o -name '*.log' -o -name '*.json' -o -name '*.txt') ; do
+             for f in $(find $PROFILING_WORKFLOW -type f -name '*.sql3') ; do
                d=$(dirname $f)
                mkdir -p $WORKSPACE/upload/profiling/$d || true
                cp -p $f $WORKSPACE/upload/profiling/$d/ || true
+               mkdir -p $WORKSPACE/upload/igprof/${CMSSW_IB}/${ARCHITECTURE}/profiling/${PROFILING_WORKFLOW}/PR-${REPORT_H_CODE}/${BUILD_NUMBER}
+               ln -s ../../../../../../pull-request-integration/PR-${REPORT_H_CODE}/${BUILD_NUMBER}/profiling/$d/$(basename $f) \
+                     $WORKSPACE/upload/igprof/${CMSSW_IB}/${ARCHITECTURE}/profiling/${PROFILING_WORKFLOW}/PR-${REPORT_H_CODE}/${BUILD_NUMBER}/$(basename $f)
              done
-             for f in $(find $PROFILING_WORKFLOW -type f -name '*.sql3' -o -name '*.log' -o -name '*.json' -o -name '*.txt') ; do
+             for f in $(find $PROFILING_WORKFLOW -type f -name '*.json' ) ; do
+               d=$(dirname $f)
+               mkdir -p $WORKSPACE/upload/profiling/$d || true
+               cp -p $f $WORKSPACE/upload/profiling/$d/ || true
+               mkdir -p $WORKSPACE/upload/profiles/$d || true
+               mkdir -p $WORKSPACE/upload/profiling/${CMSSW_IB}/${ARCHITECTURE}/${PROFILING_WORKFLOW}/PR-${REPORT_H_CODE}/${BUILD_NUMBER}
+               ln -s ../../../../../pull-request-integration/PR-${REPORT_H_CODE}/${BUILD_NUMBER}/profiling/$d/$(basename $f)  \
```

Comment:
@gartung , should not you be creating symlink pointing to 
`igprof/${CMSSW_IB}/${ARCHITECTURE}/profiling/${PROFILING_WORKFLOW}/` ? Idea was to create a symlink in pull-request-integration directory for the release. So for each PR tests there will be a symlink which points to the release igprof profiling

Reply:
The link name is on the next line after the continuation \

Reply:
but here you are creating a file (symlink) which actually points to `../../../../../pull-request-integration/PR-${REPORT_H_CODE}/${BUILD_NUMBER}/profiling/$d/$(basename $f)`  ,. I am saying that it should point to `../../../../../igprof/${CMSSW_IB}/${ARCHITECTURE}/profiling/${PROFILING_WORKFLOW}/`

Reply:
PR artifects are uploaded uploaded in to `pull-request-integration/PR-${REPORT_H_CODE}/${BUILD_NUMBER}/` directory and you want to create a symlink pointing  to the files of the release area

Reply:
The sql files need to linked under jenkins_artifcats/igprof/ and the json files need to be linked under jenkins_artifacts/profiling to be browsable with igprof_navigator and circles.

Reply:
ah right.... I got the point. Yes currently igprof_navigator has a hardcoded path.